src/model/platform-setup/setup-mac.ts

```diff
@@ -168,6 +168,7 @@ class SetupMac {
     process.env.UNITY_VERSION = buildParameters.editorVersion;
     process.env.UNITY_SERIAL = buildParameters.unitySerial;
     process.env.UNITY_LICENSING_SERVER = buildParameters.unityLicensingServer;
+    process.env.SKIP_ACTIVATION = buildParameters.skipActivation;
```

Comment:
This does indeed fix missing build param for macOS. This made me think; should we document this parameter in the docs?

Reply:
Indeed we should, there's a PR up on the docs repo, just need an approval on the PR

Reply:
PR Link: https://github.com/game-ci/documentation/pull/452